web/templates/popovers/actions_popover.hbs

```diff
@@ -7,7 +7,7 @@
                 <li class="link-item popover-menu-inner-list-item">
                     <a data-message-id="{{message_id}}" class="respond_button popover-menu-link" tabindex="0">
                         <i class="popover-menu-icon zulip-icon zulip-icon-quote-and-reply" aria-hidden="true"></i>
-                        {{t "Quote and reply" }}
+                        <span class="popover-menu-text">{{t "Quote and reply" }}</span>
```

Comment:
What's the motivation for adding these spans? I don't see CSS for `popover-menu-text` in this commit; is it just to create a container element?

Reply:
Following the new changes, `.popover-menu-text` is now being explicitly used to style the option text.

Reply:
`popover-menu-label` might be a better name for this.

Reply:
Renamed to `popover-menu-label`, thanks!